hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeStateMachine.java

```diff
@@ -701,15 +697,6 @@ private Thread getCommandHandlerThread(Runnable processCommandQueue) {
     return handlerThread;
   }
 
-  /**
-   * Returns the number of commands handled  by the datanode.
-   * @return  count
-   */
-  @VisibleForTesting
-  public long getCommandHandled() {
```

Comment:
Once this method is removed, the commandsHandled long value is just incremented in the code, but it is never read, can we remove that as well?

Reply:
Thanks for your comment and review, @fapifta .
I have updated it.

Reply:
Hi @jianghuazhu, thank you for the update, I did not meant to re-add the method again. If the method is not used anymore, let's remove it, but remove it together with the commandsHandled variable, as that does not have meaningful uses if this getter is not used.

Reply:
Thanks @fapifta  for the comment and review.
I updated it again.